vocab/systems/VOCAB_QUDT-SYSTEM-OF-UNITS-ALL-v2.1.ttl

```diff
@@ -63,9 +63,8 @@ sou:SOU_CGS
   qudt:informativeReference "http://www.tf.uni-kiel.de/matwis/amat/mw1_ge/kap_2/basics/b2_1_14.html"^^xsd:anyURI ;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/sou> ;
   rdfs:label "CGS System of Units" ;
-  rdfs:seeAlso qudt:SOU_CGS-EMU ;
-  rdfs:seeAlso qudt:SOU_CGS-ESU ;
-  rdfs:seeAlso qudt:SOU_CGS-GAUSS ;
+  rdfs:seeAlso sou:SOU_CGS-EMU ;
+  rdfs:seeAlso sou:SOU_CGS-ESU ;
```

Comment:
Why eliminate GAUSS?

Reply:
It was also showing up as undefined. Can you take a look?

Reply:
OK, added it back in.